README.md

```diff
@@ -0,0 +1,7 @@
+# 🥋 Mindset Dojo & Negotiation Mastery Program
+
+Start your journey today: build emotional wisdom, master authentic conversation, and unlock your full leadership potential at https://dojo.center/
```

Comment:
I would make this more mysterious .. 

Start your exploration at https://dojo.center